### PR TITLE
Environment Page Redesign

### PR DESCRIPTION
**Pull request by vivianliu0**
Demo: https://www.loom.com/share/562c2806a7e1494bb6b0476543512854

Demo of new changes after addressing the comments:
https://www.loom.com/share/89eec13bb03e4544a12b4dff38c70ec5

### CODE REVIEW

**Comment by vivianliu0** [comment]
@kouthar Ready for design review when you have a chance :)

File: src/Components/Admin/Rooms/roomCard.js [hunk 1]
```diff
@@ -53,94 +122,119 @@ export default function RoomCard({ data, handleEditClick, handleDeleteClick }) {
...
-            item
-            xs={12}
-            md={6}
-            lg={4}
-            spacing={1}
-            alignItems="center"
-            justify="flex-start"
-            className={classes.card}
-        >
+        <ClickAwayListener onClickAway={handleClickAway}>
```
Comment (ahmedhamodi) [inline src/Components/Admin/Rooms/roomCard.js:140]: Genuinely never used the `ClickAwayListener` library before... nice find!

File: src/Components/Admin/Rooms/roomCard.js [hunk 2]
```diff
@@ -7,43 +7,112 @@ import IconButton from "@material-ui/core/IconButton";
...
+        borderTopRightRadius: "4px",
+        borderLeft: "solid",
+        borderRight: "solid",
+        borderTop: "solid",
+        borderLeftWidth: "2px",
+        borderRightWidth: "2px",
+        borderTopWidth: "2px",
+        borderLeftColor: Colours.MainRed5,
+        borderRightColor: Colours.MainRed5,
+        borderTopColor: Colours.MainRed5,
```
Comment (ahmedhamodi) [inline src/Components/Admin/Rooms/roomCard.js:40]: Nice use of the new themes folder!

**Review by ahmedhamodi — COMMENTED** [review]
From a code perspective, this looks great.

There are issues with different screen sizes, which Vivian and I talked about offline. Created a ticket to address this as it is due to logic that was pre-existing and not due to Vivian's code.

Will test locally and see if anything odd happens.

Pending design approval. :) Good work Vivian 🎉

File: src/Components/Admin/Rooms/roomCard.js (hunk 1 above)
Comment (aaronabraham311) [reply src/Components/Admin/Rooms/roomCard.js:140]: Yeah +1!

File: src/Components/Admin/Rooms/rooms.js [hunk 3]
```diff
@@ -26,10 +31,10 @@ export default function EscapeRooms({
     return (
         <Container component={"main"} maxWidth="lg">
             <CssBaseline />
+            <Typography component="div" variant="h5" className={classes.title}>
+                Your escape rooms
+            </Typography>
```
Comment (aaronabraham311) [inline src/Components/Admin/Rooms/rooms.js:36]: I believe this should be bolded (probably h4/h3) when I look at Figma

**Review by aaronabraham311 — CHANGES_REQUESTED** [review]
Nice work as always Vivian! Couple of things:
- Can we add some top and bottom padding to the card options when you click on the three dot menu? I recommend you check out the CSS on Figma
- Can we remove the 'Admin Dashboard' text at the top?

**Comment by kouthar** [comment]
image
image
First photo: card near the bottom - bottom right aligned
Second photo: demonstrating alignment explicitly, but obvi don't show the kebab button when the menu is open lol

image
image
Third and Fourth: card near the top/middle - top right aligned

File: src/Components/Admin/Rooms/rooms.js (hunk 3 above)
Comment (vivianliu0) [reply src/Components/Admin/Rooms/rooms.js]: Thanks, updated and bolded the title. h4/h3 increase in font sizes by quite a bit compared to h5, I think h5 is the closest fit
(image of h4)
image

**Comment by vivianliu0** [comment]
Upd

> Nice work as always Vivian! Couple of things:
> 
> * Can we add some top and bottom padding to the card options when you click on the three dot menu? I recommend you check out the CSS on Figma
> * Can we remove the 'Admin Dashboard' text at the top?

Updated, you can check out the second loom video I added to the PR to see the changes

**Comment by vivianliu0** [comment]
> image
> image
> First photo: card near the bottom - bottom right aligned
> Second photo: demonstrating alignment explicitly, but obvi don't show the kebab button when the menu is open lol
> 
> image
> image
> Third and Fourth: card near the top/middle - top right aligned

**Review by aaronabraham311 — APPROVED** [review]
LGTM!

**Review by ahmedhamodi — APPROVED** [review]
lgtm

**Pull request merged**